velox/exec/tests/QueryAssertionsTest.cpp

```diff
@@ -493,4 +494,23 @@ TEST_F(QueryAssertionsTest, intervalDayTime) {
   assertQuery(plan, "SELECT * FROM tmp");
 }
 
+TEST_F(QueryAssertionsTest, equalPlans) {
```

Comment:
equalPlans test name confuses me; I expect the test to verify that 2 plans are equal, not that they produce same results

BTW, can we also add a test case where assertEqualResults returns false?

Reply:
done.